### ISSUE TITLE
[BUG] CLI can't add the second repo

### ISSUE THREAD

**Issue by kurotu** [vrchat-community/creator-companion#37]
**Describe the bug**
`vpm add repo` can't add the second repo.

**To Reproduce**
1. Make sure that `userRepos` is blank array in VCC's settings.json.
2. Add the first repo by `vpm add repo https://foo.com/...`
3. Try to add another repo by `vpm add repo https://bar.com/...`
4. See error:
    ```
    [23:26:41 ERR] You already have a repo with the url https://bar.com/...
    ```
5. `userRepos` contains the first repo only. But Repos folder contains two user repo json files.

**Expected behavior**
`vpm add repo` can add the second repo as well as the first repo.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version: 21H2

**Additional context**
VPM CLI version: 0.1.12

**Comment by orels1** [comment]
Addressed internally as a part of the Community Listings work

**Comment by orels1** [comment]
This has been fixed as a part of 2.1.0
